src/ml_tooling/storage/artifactory.py

```diff
@@ -72,14 +72,14 @@ def get_list(self) -> List["ArtifactoryPath"]:
 
         return sorted(self.artifactory_path.glob("*/*.pkl"))
 
-    def load(self, filename: Pathlike) -> Estimator:
+    def load(self, file_path: Pathlike) -> Estimator:
```

Comment:
Does this still work with return values from `storage.list_estimators`? Is there a test for that?

Reply:
No it didnt and we didnt have a test for it either :) added it in https://github.com/andersbogsnes/ml_tooling/pull/346/commits/56cdb8c8e205423beff6e5ae02ade4c8558fc42d